### ISSUE TITLE
Table: Sorting icon should indicate sort direction

### ISSUE THREAD

**Issue by ChrisKnightLDN** [Greater-London-Authority/ldn-viz-tools#469]
![Screenshot 2024-08-01 at 16 52 46](https://github.com/user-attachments/assets/a8616d67-c98f-46a0-ba77-d8b398982c6f)

**Comment by jamesscottbrown** [comment]
Yes, it should use unambiguous sort-direction icons like those in Font Awesome (`sortAlphaUp`, `sortAlphaDown`, `sortAmountUp`, `sortAmountDown`)/Octicons (`sortAsc`, `sortDesc`)/ Boostrap (`sortAlphaUp`, `sortAlphaDown`, `sortDown`, `sortUp`)/Flat Color (`AlphabeticalSortingAz`, `AlphabeticalSortingAz`, `NumericalSorting12`, `NumericalSorting21`)/IcoMoon (`SortNumbericDesc`, `SortNumbericAsc`, `SortAlphaAsc`, `SortAlphaDsc`) /Line Awesome (`SortAlphaDownSolid`, `SortAlphaUpSolid`, `SortAmountDownSolid`, `SortAmountUpSolid`) - not just a simple up-down arrow.

Heroicons only has `arrow-up`/`arrow-down` and `chevron-up`/`chevron-down` icons, so we'll need to use something from outside this set (either borrowed from a different icon library or drawn custom).

**Comment by ChrisKnightLDN** [comment]
Heroicons bars-arrow-up, bars-arrow-down
